commands/get-search-terms.ts

```
@@ -99,18 +104,20 @@ async function getSearchTermsCommand(options: SearchTermsOptions): Promise<void>
         console.log('');
 
         let totalViews = 0;
+        let totalEngagedViews = 0;
```

Biome (2.5.7)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)